docs/Installing.rst

```diff
@@ -101,56 +96,29 @@ DFHack binary and system data is stored, a ``dfhack-config`` folder where user
 of Dwarf Fortress may place DF in another folder, so ensure that the ``hack``
 folder ends up next to the ``data`` folder.
 
-.. note::
-
-    On Windows, installing DFHack will overwrite ``SDL.dll``. This is
-    intentional and necessary for DFHack to work, so be sure to choose to
-    overwrite ``SDL.dll`` if prompted. (If you are not prompted, you may be
-    installing DFHack in the wrong place.)
-
-
 Uninstalling DFHack
 ===================
 
-Uninstalling DFHack essentially involves reversing what you did to install
-DFHack. On Windows, replace ``SDL.dll`` with ``SDLreal.dll`` first. Then, you
-can remove any files that were part of the DFHack archive. DFHack does not
-currently maintain a list of these files, so if you want to completely remove
-them, you should consult the DFHack archive that you installed for a full list.
-Generally, any files left behind should not negatively affect DF.
+To disable DFHack, delete or rename the "dfhooks" library. This is
+``dfhooks.dll`` on Windows, ``libdfhooks.so`` on Linux, or ``libdfhooks.dylib``
+on OSX.
 
+You can remove any other files that were part of the DFHack archive, but you
+don't strictly need to. DFHack does not currently maintain a list of these
+files, so if you want to completely remove them, you should consult the DFHack
+archive that you installed for a full list. Leftover files should not negatively
+affect your game.
 
 Upgrading DFHack
 ================
 
-The recommended approach to upgrade DFHack is to uninstall DFHack first, then
-install the new version. This will ensure that any files that are only part
-of the older DFHack installation do not affect the new DFHack installation
-(although this is unlikely to occur).
-
-It is also possible to overwrite an existing DFHack installation in-place.
-To do this, follow the installation instructions above, but overwrite all files
-that exist in the new DFHack archive (on Windows, this includes ``SDL.dll`` again).
-
-.. note::
-
-    You may wish to make a backup of your ``dfhack-config`` folder first if you
-    have made changes to it. Some archive managers (e.g. Archive Utility on macOS)
-    will overwrite the entire folder, removing any files that you have added.
-
-
-Pre-packaged DFHack installations
-=================================
-
-There are :wiki:`several packs available <Utility:Lazy_Newb_Pack>` that include
-DF, DFHack, and other utilities. If you are new to Dwarf Fortress and DFHack,
-these may be easier to set up. Note that these packs are not maintained by the
-DFHack team and vary in their release schedules and contents. Some may make
-significant configuration changes, and some may not include DFHack at all.
+The recommended approach to upgrade DFHack is to remove the ``hack`` folder,
+then install the new version. This will ensure that any files that are only part
+of the older DFHack installation do not affect the new DFHack installation.
```

Comment:
We should keep the note about preserving dfhack-config - that is still relevant

Reply:
Oh, I just saw your comment at https://github.com/DFHack/dfhack/issues/2326#issuecomment-1371273593. That's fair, but I think removing the note should be done after that change is made (or alongside it).

Reply:
Fair enough, even after we remove the directory from the archive and this specific issue goes away, we should still note that that's where all the user-written data is (apart from blueprints and exported stockpile settings, but those might move in someday too)

Reply:
note that this PR won't be merged until we have a new DF binary in our hands -- otherwise it would break current dev. The dfhack-config PR will be merged before this one.

Reply:
> We should keep the note about preserving dfhack-config - that is still relevant

done, but moved to the `Uninstalling` section where it is now more relevant.